modules/programs/abook.nix

```diff
@@ -0,0 +1,55 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.programs.abook;
+
+in {
+  options.programs.abook = {
+    enable = mkEnableOption "Abook";
+
+    extraConfig = mkOption {
+      type = types.lines;
+      default = "";
+      example = ''
```

Comment:
Would it be possible to shorten the example? For example, removing the comments and only keeping one of the `field` and `view` entries each? That is, to make it minimal but still giving an idea of the things you can put here.